pandas/tests/scalar/timestamp/test_arithmetic.py

```diff
@@ -7,11 +7,12 @@
 import pandas.util.testing as tm
 from pandas.compat import long
 from pandas.tseries import offsets
+from pandas.tseries.frequencies import to_offset
 from pandas import Timestamp, Timedelta
 
 
 class TestTimestampArithmetic(object):
-    def test_overflow_offset(self):
+    def test_overflow_offset1(self):
```

Comment:
Can you keep the original name of this test.

Reply:
Split up the tests per the other comment..